.github/workflows/recurring-release-prod.yml

```diff
@@ -0,0 +1,18 @@
+name: "Recurring release: Prod"
+on:
+  # Can be manually triggered
+  workflow_dispatch:
+  # Runs automatically every 30 minutes from 8am to 8pm Monday to Friday.
+  # This currently UTC -> EDT.
+  schedule:
+    - cron: "*/30 0,12-23 * * 1-5"
```

Comment:
This is modified from the original timeframe to shift it to running 8am to 8pm ET.